modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc

```diff
@@ -10,24 +10,21 @@
 == Overview
 
 
-{cbl-te}_'s_ support for pre-built databases means you can pre-load your app with data instead of syncing it down from {sgw-te} during startup.
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application. 
 
-Avoiding an initial bulk sync will help reduce startup time and network transfer costs, leading to a better consumer experience.
-It is typically more efficient to download bulk data using the http/ftp stream  employed during the application installation than to install a smaller application bundle and then have to use a replicator to pull-in the bulk data.
+Avoiding an initial bulk sync reduces startup time and network transfer costs. It is typically more efficient to download bulk data using the http/ftp stream employed during the application installation than to install a smaller application bundle and then use a replicator to pull-in the bulk data.
```

Comment:
new line for new sentence

Reply:
(again sorry, that's not something you introduced, but just as you edited it 😄 anyway, you get the point)